arielperez-app/src/main/java/edu/arielperez/advancedjava/application/StockQuoteApplication.java

```diff
@@ -45,12 +60,35 @@ public static void main(String[] args) {
 			System.out.println(price.toString());
 		}
 
-		// Get price now
-		StockQuote testStockPrice = stockServiceImplementation.getQuote(symbol);
+		// Calls the getQuote method to pull price information between two dates for a
+		// stock symbol
+		List<StockQuote> listWeeklyPrices = stockServiceImplementation.getQuote(symbol, fromDate, toDate,
+				IntervalEnum.WEEKLY);
+
```

Comment:
break up the main method into pieces (individual methods) ; this is too long as it is.